content/reference/sql/version-control/dolt-sql-procedures.md

```diff
@@ -487,6 +488,60 @@ CALL DOLT_ADD('table')
 CALL DOLT_RESET('table')
 ```
 
+
+## `DOLT_REMOTE()`
+
+Adds a remote for a repository at given url, or removes an existing remote with its remote-tracking branches 
+and configuration settings. Works exactly like [`dolt remote`](../../cli.md#dolt-remote) on the CLI, and takes 
+the same arguments except for listing tags. To list existing tags, use 
+[`dolt_remotes`](./dolt-system-tables.md#dolt_remotes) system table.
+
+```sql
+
+CALL DOLT_REMOTE('add','remote_name','remote_url');
+CALL DOLT_REMOTE('remove','existing_remote_name');
+```
+
+### Options
+
+`--aws-region=<region>`: Specify a cloud provider region associated with this remote.
+
+`--aws-creds-type=<creds-type>`: Specify a credential type.  Valid options are role, env, and file.
+
+`--aws-creds-file=<file>`: Specify an AWS credentials file.
+
+`--aws-creds-profile=<profile>`: Specify an AWS profile to use.
+
+### Example
+
+```sql
+-- Add a HTTP remote 
+CALL DOLT_REMOTE('add','origin','https://doltremoteapi.dolthub.com/Dolthub/museum-collections');
+
+-- Add a filesystem based remote
+CALL DOLT_REMOTE('add','origin2','file:///Users/jennifer/datasets/museum-collections');
+
```

Comment:
I love the dolt remote URL and file URL examples. Might be worth adding a DoltHub `<org>/<database>` example, too, since that's a neat shorthand syntax we support.